app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,29 @@
+class BookingsController < ApplicationController
+  # def new
+  #   @artwork = Artwork.find(params[:artwork_id])
+  #   @booking = @artwork.bookings.new
+  #   @booking.user = current_user
+  # end
+
+  def create
+    @artwork = Artwork.find(params[:artwork_id])
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+    @booking.artwork = @artwork
+    raise
```

Comment:
Thank you Issam all looks really good , only think that I want to ask you for this Raise for debug? Is it is could you delete and pull request one more time?

Reply:
Okay. I will do it now.